lib/views/spinner/index.ios.js

```diff
@@ -8,7 +8,7 @@ var React = require('react');
 
 var {
   ActivityIndicatorIOS,
-  Component,
+  Component, // eslint-disable-line no-unused-vars
```

Comment:
My bad, it was removed in latest master but did not take it correctly in the rebase. Previous to that this was failing due to unused vars, and thought it could not be removed that is why I commented the line. 
Removed now.